src/openrct2/GameLimits.h

```diff
@@ -0,0 +1,19 @@
+/*****************************************************************************
```

Comment:
Calling it GameLimits.h is too broad - there are dozens, if not hundreds of such limits. I would call it `src/openrct2/drawing/ScrollingText.h`.

Reply:
The purpose of this file is to be precisely that, broad. We should move all technical limits in there, right now its way too scattered.

Reply:
I have to disagree here .What we’ll end up with is a mega file that gets impossible to read. There are hundreds of limits.

Reply:
If there are so many limits its even more reason to have a central location for them and not have hundreds of constants scattered in my opinion.

Reply:
It will only complicate matters and make every part of the code tied to it IMO.